dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/HasDex.scala

```diff
@@ -3,14 +3,15 @@ package com.wavesplatform.dex.it.api.dex
 import java.util.concurrent.ThreadLocalRandom
 
 import cats.Functor
+import cats.instances.FutureInstances
 import com.typesafe.config.{Config, ConfigFactory}
 import com.wavesplatform.dex.it.api.BaseContainersKit
 import com.wavesplatform.dex.it.docker.base.info.DexContainerInfo
 import com.wavesplatform.dex.it.docker.base.{BaseContainer, DexContainer}
 import com.wavesplatform.dex.it.fp.CanExtract
 import mouse.any._
 
-trait HasDex { self: BaseContainersKit =>
+trait HasDex extends FutureInstances { self: BaseContainersKit =>
```

Comment:
Why do we need this? Does this reduce imports?

Reply:
Yes, it reduces amount of imports for asyncApi, when we need a functor for Future. Moved `extends` to `MatcherSuiteBase`, it will be useful for NODE's API too.